src/main/java/org/mitre/synthea/export/Exporter.java

```diff
@@ -287,4 +295,13 @@ public static String filename(Person person, String extension) {
           + extension;
     }
   }
+
+  public static String filename2(Person person, String encounterNumber, String extension) {
```

Comment:
`filename2` is not a good name (try not to ever use numbers in a function name) -- try something more descriptive like `filename_per_encounter(...)` or something.